<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookManageWindow</class>
 <widget class="QDialog" name="BookManageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书维护</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidgetBooks">
     <column>
      <property name="text">
       <string>书号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出版社</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出版日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>库存</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>备注</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayoutForm">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBookId">
       <property name="text">
        <string>书号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditBookId"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelBookName">
       <property name="text">
        <string>书名：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditBookName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAuthor">
       <property name="text">
        <string>作者：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditAuthor"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelPublisher">
       <property name="text">
        <string>出版社：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEditPublisher"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPublishDate">
       <property name="text">
        <string>出版日期：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateEditPublishDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="labelStock">
       <property name="text">
        <string>库存：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinBoxStock">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxStatus"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="labelRemark">
       <property name="text">
        <string>备注：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QTextEdit" name="textEditRemark">
       <property name="maximumHeight">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpdate">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDelete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonResetForm">
       <property name="text">
        <string>清空表单</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
